checkov/kubernetes/runner.py

```diff
@@ -228,19 +228,38 @@ def get_skipped_checks(entity_conf):
         if "metadata" in entity_conf.keys():
             metadata = entity_conf["metadata"]
     if "annotations" in metadata.keys() and metadata["annotations"] is not None:
-        for key in metadata["annotations"].keys():
-            skipped_item = {}
-            if "checkov.io/skip" in key or "bridgecrew.io/skip" in key:
-                if "CKV_K8S" in metadata["annotations"][key]:
-                    if "=" in metadata["annotations"][key]:
-                        (skipped_item["id"], skipped_item["suppress_comment"]) = metadata["annotations"][key].split("=")
+        if isinstance(metadata["annotations"], dict):
```

Comment:
@tronxd LGTM, can we use the force_list method to avoid the code duplication?

Reply:
Done